jre_emul/Makefile

```diff
@@ -206,8 +206,8 @@ dist_includes: $(BUILD_DIR)/.jre_dist_includes
 $(BUILD_DIR)/.jre_dist_includes: $(PUBLIC_TRANSLATED_HEADERS:%=$(GEN_OBJC_DIR)/%) \
     $(PUBLIC_EMULATION_HEADERS:%=$(EMULATION_CLASS_DIR)/%) | $(DIST_INCLUDE_DIR)
 	@echo copying jre_emul header files
-	@(cd $(EMULATION_CLASS_DIR) && tar cf - \
-	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%)) \
+	@cd $(EMULATION_CLASS_DIR) && tar cf - \
```

Comment:
When I try this change locally, I'm getting "./dist/include: No such file or directory". The pipe has higher precedence than the "&&" so the initial change directory is not run in a subshell.

Could you try switching the "&&" with a ";" while preserving the subshell brackets. Just to see if it works for you. Another thing to try is pushd and popd commands.